ddp-workspace/projects/ddp-pancan/src/app/components/anchors-page/anchors-page.component.html

```diff
@@ -0,0 +1,43 @@
+<main class="page">
+    <h2 class="title static-page-title">{{source + '.Title' | translate}}</h2>
+
+    <nav class="anchors">
+        <ul>
+            <li *ngFor="let section of (source + '.Sections') | translate; let i = index;" class="anchor">
+                <a [routerLink]="[routerLink]"  [fragment]="'section' + i"
+                   (click)="scrollToAnchor('section' + i)">{{section.Title}}
+                </a>
+            </li>
+        </ul>
+    </nav>
+
+    <section  *ngFor="let section of (source + '.Sections') | translate; let i = index;" class="section">
+        <div [id]="anchorLabel + i"
+             class="section__content"
+             [class.column-direction]="section.PeopleList?.length">
+            <h2 class="section__title static-page-title">{{section.Title}}</h2>
```

Comment:
should it be h3? since the page title already has h2

Reply:
Agree, fixed.